### ISSUE TITLE
API Review

### ISSUE THREAD

**Issue by plfiorini** [hawaii-desktop/qtaccountsservice#7]
Review the API.

**Comment by plfiorini** [comment]
See #9

**Comment by plfiorini** [comment]
@xiangzhai Anyone willing to help with API review is welcome here :)

**Comment by plfiorini** [comment]
For example KDE developers

**Comment by xiangzhai** [comment]
Yes, David Edmundson, who concerned by the blocking calls in AccountsService, is the one ;-)

**Comment by xiangzhai** [comment]
I will email to plasma-devel mailing list and cc those projects` core developers and maintainers, they will show use what they concern ;-)
